presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkPageOutputOperator.java

```diff
@@ -185,6 +185,7 @@ public void addInput(Page page)
                 .collect(toImmutableList());
 
         serializedPages.forEach(outputBuffer::enqueue);
+        operatorContext.recordOutput(page.getSizeInBytes(), page.getPositionCount());
```

Comment:
Are these existing bugs or something? @pgupta2 could you help to verify?

Reply:
This is going to record the output bytes/rows for data that is being shuffled, which is currently missing from operator stats in PoS. Nice catch!!